lua/telescope/previewers/buffer_previewer.lua

```diff
@@ -182,8 +182,8 @@ local handle_file_preview = function(filepath, bufnr, stat, opts)
       end
     end
 
+    local mb_filesize = math.floor(stat.size / bytes_to_megabytes)
```

Comment:
hmm, why did we every floor this?

It doesnt make sense to me, so e.g. 821792byte file ~ 0.78 mb, after the floor its 0.
seems weird. so i can never actually ignore all files >= 0.5mb e.g.

also if we now have a file with 1500000byte ~ 1.43mb, after the floor its 1. which is in fact not > highlight_limit an will be highlighted. so we actually start ignoring files > 2 not > 1. 

"The maximum file size in MB attempted to be highlighted." suggest that 1mb is the maximum filesize that is being highlighted, but 1.2mb files will still get highlighting.

I know the floor was already there (so not your mistake), but in my opinion it should not be floored.

Any thoughts?

Reply:
You're right, let's use this opportunity to change it.

I just updated the PR.